charts/newrelic-infrastructure/templates/ksm/_tolerations_helper.tpl

```diff
@@ -0,0 +1,25 @@
+{{- /*
+Defaults for ksm while keeping then overridable.
+*/ -}}
+{{- define "nriKubernetes.ksm.tolerations.defaults" -}}
+- operator: "Exists"
+  effect: "NoSchedule"
+- operator: "Exists"
+  effect: "NoExecute"
+{{- end -}}
+
+
+
+{{- /*
+As this chart deploys what it should be three charts to maintain the transition to v3 as smooth as possible.
+This means that this chart has 3 tolerations so a helper should be done per scraper.
+*/ -}}
+{{- define "nriKubernetes.ksm.tolerations" -}}
+{{- if gt (len .Values.ksm.tolerations) 0  -}}
```

Comment:
Because it is an array and I wanted to test that it is an array nobody knows why now and make Helm to yell an error. 

I have tested it and it is not a nice message... Fixed in [#c917114](https://github.com/newrelic/nri-kubernetes/pull/383/commits/c91711476f8b554deeb65fb1c9b628ffb59a4770)